git.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+lastUpdate= "there is no file"
+INIT(){
+    if[-d /home/git];then
+        echo "git 目录存在,初始化git"
+        git init /home/git
+    else
+        echo "git 目录不存在，创建目录"
+        mkdir /home/git
+        git init /home/git
+    fi
```

Comment:
```suggestion
    if [ -d /home/git ]; then
        echo "git 目录存在,初始化git"
    else
        echo "git 目录不存在，创建目录"
        mkdir /home/git
    fi
```
1. shell 的参数划分是通过空格来进行的，包括 if 的使用。if 后面其实是填写一个语句，然后 if 用这个语句的返回值来作为运行哪个分支的依据
2. `[ ]` 这一看似是 shell 语法的东西其实是一个名为 `[` 的内部命令（详见 `man [`），它的不同参数之间需要以空格分开，最后一个参数需要是 `]` 来形成一种视觉上的直观，此处 `-d`、`/home/git`、`]` 之间是不同的参数；同时 `[` 会以判断结果作为返回值。
3. 此处 `/home/git` 似乎不需要是一个 git 仓库（如果我遗漏了什么请提醒我）